snappy/click.go

```diff
@@ -316,12 +326,12 @@ func (m *packageYaml) addPackageServices(baseDir string, inhibitHooks bool, inte
 		}
 		// If necessary, generate the DBus policy file so the framework
 		// service is allowed to start
-		if m.Type == snap.TypeFramework && service.BusName != "" {
-			content, err := genBusPolicyFile(service.BusName)
+		if m.Type == snap.TypeFramework && app.BusName != "" {
+			content, err := genBusPolicyFile(app.BusName)
```

Comment:
The organization here needs much improvement. The structure that holds metadata for a package  (this is a _snapYaml!_) should never be dealing with security by itself.